### PR TITLE
Happy Thoughts - Martin

### PR DESCRIPTION
**Pull request by Martin-Joensson**
## Netlify link
https://manyhappythoughts.netlify.app/

### CODE REVIEW

**Review by dzc1 — APPROVED** [review]
Grat project Martin 💪💪💪 

Loving to see the simplicity yet effectiveness of your syntax and it seems that you are grasping the full power of the react ecosystem. Destructuring comps as well provides a smooth review. Keep up the amazing work Martin!

File: src/index.css [hunk 1]
```diff
@@ -7,7 +7,140 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
+* {
+  --clr_background: white;
+  --clr_form_background: #f2f0f0;
+  --clr_heart_button: #ffadad;
+  --clr_heart_button_hover: #efefef;
+  --clr_heart_button_active: grey;
```
Comment (Caracal23) [inline src/index.css:15]: Great job in creating and assingning CSS classes 👏🏻

File: src/components/Time.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+
+export const Time = (props) => {
+  const now = new Date();
```
Comment (Caracal23) [inline src/components/Time.jsx:4]: Good job with the logic for calculating the time stam; I used another approach with the Dependency of "moment". Maybe you find it useful next time 😀

**Review by Caracal23 — COMMENTED** [review]
Martin, your code is well organized, easy to follow/read. Great use on the knowledge learn 🚀

**Pull request closed without merging**